algebras/algebra/src/main/scala/endpoints4s/algebra/Urls.scala

```diff
@@ -108,6 +108,22 @@ trait Urls extends PartialInvariantFunctorSyntax {
       value: QueryStringParam[A]
   ): QueryString[A]
 
+  /** Builds a `QueryString` with one optional parameter, which has a default value.
+    *
+    * Examples:
+    *
+    * {{{
+    *   qsWithDefault[Int]("page", 1) // optional `page` parameter, with default value 1
+    * }}}
+    *
+    * @param name Parameter’s name
+    * @tparam A Type of the value carried by the parameter
+    * @group operations
+    */
+  def qsWithDefault[A](name: String, default: A, docs: Documentation = None)(implicit
```

Comment:
I tried to add this method as the `qs` overload, but compiler complained about multiple overloads with default values. So, I named this one differently

Reply:
Thanks to [this great comment](https://github.com/endpoints4s/endpoints4s/pull/879#discussion_r687485809), I added a default implementation to algebra, which should solve compatibility issues and make this change much simpler in general.
I'm not sure why the CI fails for `versionPolicyCheck`, though, I don't have any issues running it locally. Would a squash help maybe?

Reply:
CI error disappeared after rebase.
Note I added `versionPolicyIntention := Compatibility.None` to `openapi` module build definition, as I understand that's the thing to do before the next release when there is a incompatible change in interpreter.